### PR TITLE
fixes from pr

### PR DESCRIPTION
**Pull request by leralyshls**
logo fixes from pr

### CODE REVIEW

File: src/components/logo/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import logoSvg from '../../assets/images/logo.svg';
+import { LogoContainer, LogoImg } from './style';
+
+export interface LogoProps {
+  height?: number;
+}
+
+const Logo = ({ height }: LogoProps) => {
+  return (
+    <LogoContainer height={height || undefined}>
```
Comment (avivahls) [inline src/components/logo/Logo.tsx:11]: why you need the undefined?

File: src/components/logo/style.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import { LogoProps } from './Logo';
+
+export const LogoContainer = styled.div`
+  ${(props: LogoProps) => `
+  height: ${props.height ? `${props.height}rem` : '50px'};
```
Comment (avivahls) [inline src/components/logo/style.ts:6]: remove px

Comment (leralyshls) [reply src/components/logo/style.ts]: removed

Comment (avivahls) [reply src/components/logo/style.ts]: you still have '50px'

**Review by avivahls — APPROVED** [review]

**Pull request merged**